### ISSUE TITLE
error

### ISSUE THREAD

**Issue by razzietheman** [sabeechen/hassio-google-drive-backup#229]
###### Description:
```
If you have anything else that could help explain what happened, click "Markdown" above and write it here.
```

 Addon version: `0.101.1`
 Home Assistant Version: `0.114.0`
 Supervisor Version: `235`
 Architecture: `armv7`
 Date: `2020-08-20T09:22:58.048371+00:00`
 Timezone: `Europe/Stockholm`
 Failure Time: `2020-08-20T09:08:28.306239+00:00`
 Last Good Sync: `2020-08-20T09:08:27.922211+00:00`
 ###### Exception:
 ```
 
  addon/backup/model/coordinator.py:148 (_sync)
  addon/backup/model/model.py:162 (sync)
  addon/backup/drive/drivesource.py:161 (save)
  addon/backup/drive/drivesource.py:148 (save)
  addon/backup/drive/driverequests.py:200 (create)
  addon/backup/drive/driverequests.py:322 (retryRequest)
  addon/backup/creds/driverequester.py:39 (request)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 410, message='Gone', url=URL('https://www.googleapis.com/upload/drive/v3/files/?uploadType=resumable&supportsAllDrives=true&upload_id=AAANsUm4Si05qnQYnob6wjzOv_arjOc3tKYBYsVFrC7wJw20wDCEJGjJxxGjxY8A7fxDPJ9oeP3KVdOtBDQa3pCL5A')

 ```
 Snapshots:
 ```
 {
    "HomeAssistant": {
        "snapshots": 3,
        "retained": 0,
        "deletable": 3,
        "name": "HomeAssistant",
        "latest": "2020-08-20T06:15:44Z",
        "size": "81.3 MB"
    },
    "GoogleDrive": {
        "snapshots": 3,
        "retained": 0,
        "deletable": 3,
        "name": "GoogleDrive",
        "latest": "2020-08-19T06:06:34Z",
        "size": "78.2 MB"
    }
}
 ```
 ###### Config:
 ```
 {
    "Setting.MAX_SNAPSHOTS_IN_HASSIO": 3,
    "Setting.MAX_SNAPSHOTS_IN_GOOGLE_DRIVE": 3,
    "Setting.DAYS_BETWEEN_SNAPSHOTS": 1.0,
    "Setting.SNAPSHOT_NAME": "REDACTED",
    "Setting.SNAPSHOT_TIME_OF_DAY": "08:00",
    "Setting.SEND_ERROR_REPORTS": true,
    "Setting.BACKGROUND_COLOR": "#000000",
    "Setting.ACCENT_COLOR": "#3a4e57"
}
 ```
 ###### Addon Logs:
 ```
 08-20 11:08:22 DEBUG [backup.worker.watcher] Backup directory modified event
08-20 11:08:22 DEBUG [backup.worker.watcher] Backup directory changed
08-20 11:08:22 DEBUG [backup.worker.watcher] Backup directory moved event
08-20 11:08:22 DEBUG [backup.worker.watcher] Backup directory changed
08-20 11:08:22 DEBUG [backup.worker.watcher] Backup directory modified event
08-20 11:08:22 DEBUG [backup.worker.watcher] Backup directory changed
08-20 11:08:22 DEBUG [backup.worker.watcher] Backup directory modified event
08-20 11:08:22 DEBUG [backup.worker.watcher] Backup directory changed
08-20 11:08:22 DEBUG [backup.worker.watcher] Backup directory deleted event
08-20 11:08:22 DEBUG [backup.worker.watcher] Backup directory changed
08-20 11:08:22 DEBUG [backup.worker.watcher] Backup directory modified event
08-20 11:08:23 DEBUG [backup.model.syncer] Sync requested by Backup Directory Watcher
08-20 11:08:23 INFO [backup.model.coordinator] Syncing Snapshots
08-20 11:08:23 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/snapshots
08-20 11:08:23 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/files/?q=%271PZqv46O0K7BA0OgiUblTkLlXqout20SZ%27+in+parents&fields=nextPageToken%2Cfiles%28id%2Cname%2CappProperties%2Csize%2Ctrashed%2CmimeType%2CmodifiedTime%2Ccapabilities%2Cparents%29&pageSize=100&supportsAllDrives=true&includeItemsFromAllDrives=true&corpora=allDrives
08-20 11:08:23 INFO [backup.drive.drivesource] Uploading 'Full Snapshot 2020-08-20' to Google Drive
08-20 11:08:23 DEBUG [backup.drive.driverequests] Attempting to resume a previosuly failed upload where we left off
08-20 11:08:23 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/upload/drive/v3/files/?uploadType=resumable&supportsAllDrives=true&upload_id=AAANsUm4Si05qnQYnob6wjzOv_arjOc3tKYBYsVFrC7wJw20wDCEJGjJxxGjxY8A7fxDPJ9oeP3KVdOtBDQa3pCL5A
08-20 11:08:23 ERROR [backup.model.coordinator] 
  addon/backup/model/coordinator.py:148 (_sync)
  addon/backup/model/model.py:162 (sync)
  addon/backup/drive/drivesource.py:161 (save)
  addon/backup/drive/drivesource.py:148 (save)
  addon/backup/drive/driverequests.py:200 (create)
  addon/backup/drive/driverequests.py:322 (retryRequest)
  addon/backup/creds/driverequester.py:39 (request)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 410, message='Gone', url=URL('https://www.googleapis.com/upload/drive/v3/files/?uploadType=resumable&supportsAllDrives=true&upload_id=AAANsUm4Si05qnQYnob6wjzOv_arjOc3tKYBYsVFrC7wJw20wDCEJGjJxxGjxY8A7fxDPJ9oeP3KVdOtBDQa3pCL5A')

08-20 11:08:23 INFO [backup.model.coordinator] I'll try again in an hour
08-20 11:08:27 INFO [backup.worker.watcher] Backup directory changed
08-20 11:08:27 DEBUG [backup.model.syncer] Sync requested by Backup Directory Watcher
08-20 11:08:27 INFO [backup.model.coordinator] Syncing Snapshots
08-20 11:08:27 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/snapshots
08-20 11:08:27 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/files/?q=%271PZqv46O0K7BA0OgiUblTkLlXqout20SZ%27+in+parents&fields=nextPageToken%2Cfiles%28id%2Cname%2CappProperties%2Csize%2Ctrashed%2CmimeType%2CmodifiedTime%2Ccapabilities%2Cparents%29&pageSize=100&supportsAllDrives=true&includeItemsFromAllDrives=true&corpora=allDrives
08-20 11:08:28 INFO [backup.drive.drivesource] Uploading 'Full Snapshot 2020-08-20' to Google Drive
08-20 11:08:28 DEBUG [backup.drive.driverequests] Attempting to resume a previosuly failed upload where we left off
08-20 11:08:28 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/upload/drive/v3/files/?uploadType=resumable&supportsAllDrives=true&upload_id=AAANsUm4Si05qnQYnob6wjzOv_arjOc3tKYBYsVFrC7wJw20wDCEJGjJxxGjxY8A7fxDPJ9oeP3KVdOtBDQa3pCL5A
08-20 11:08:28 ERROR [backup.model.coordinator] 
  addon/backup/model/coordinator.py:148 (_sync)
  addon/backup/model/model.py:162 (sync)
  addon/backup/drive/drivesource.py:161 (save)
  addon/backup/drive/drivesource.py:148 (save)
  addon/backup/drive/driverequests.py:200 (create)
  addon/backup/drive/driverequests.py:322 (retryRequest)
  addon/backup/creds/driverequester.py:39 (request)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 410, message='Gone', url=URL('https://www.googleapis.com/upload/drive/v3/files/?uploadType=resumable&supportsAllDrives=true&upload_id=AAANsUm4Si05qnQYnob6wjzOv_arjOc3tKYBYsVFrC7wJw20wDCEJGjJxxGjxY8A7fxDPJ9oeP3KVdOtBDQa3pCL5A')

08-20 11:08:28 INFO [backup.model.coordinator] I'll try again in an hour
 ```
 ###### Supervisor Logs:
 ```
 [32m20-08-20 09:08:28 INFO (MainThread) [supervisor.api.security] /snapshots/d9ef4ed8/download access from cebe7a76_hassio_google_drive_backup[0m
[32m20-08-20 09:08:28 INFO (MainThread) [supervisor.api.snapshots] Download snapshot d9ef4ed8[0m
[32m20-08-20 09:08:31 INFO (MainThread) [supervisor.homeassistant] Update Home Assistant to version 0.114.2[0m
[32m20-08-20 09:08:31 INFO (SyncWorker_4) [supervisor.docker.interface] Update image homeassistant/raspberrypi3-homeassistant:0.114.0 to homeassistant/raspberrypi3-homeassistant:0.114.2[0m
[32m20-08-20 09:08:31 INFO (SyncWorker_4) [supervisor.docker.interface] Pull image homeassistant/raspberrypi3-homeassistant tag 0.114.2.[0m
[32m20-08-20 09:11:44 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/sabeechen/hassio-google-drive-backup repository[0m
[32m20-08-20 09:11:44 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/home-assistant/hassio-addons repository[0m
[32m20-08-20 09:11:44 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/hassio-addons/repository repository[0m
[32m20-08-20 09:11:58 INFO (MainThread) [supervisor.store] Load add-ons from store: 67 all - 0 new - 0 remove[0m
[32m20-08-20 09:17:18 INFO (MainThread) [supervisor.homeassistant] Updated Home Assistant API token[0m
[32m20-08-20 09:17:56 INFO (SyncWorker_4) [supervisor.docker.interface] Stop homeassistant application[0m
[32m20-08-20 09:18:24 INFO (MainThread) [supervisor.host.info] Update local host information[0m
[32m20-08-20 09:18:24 INFO (MainThread) [supervisor.utils.gdbus] Call org.freedesktop.DBus.Properties.GetAll on /org/freedesktop/hostname1[0m
[32m20-08-20 09:18:25 INFO (MainThread) [supervisor.host.services] Update service information[0m
[32m20-08-20 09:18:25 INFO (MainThread) [supervisor.utils.gdbus] Call org.freedesktop.systemd1.Manager.ListUnits on /org/freedesktop/systemd1[0m
[32m20-08-20 09:18:25 INFO (MainThread) [supervisor.host.network] Update local network DNS information[0m
[32m20-08-20 09:18:25 INFO (MainThread) [supervisor.utils.gdbus] Call org.freedesktop.DBus.Properties.GetAll on /org/freedesktop/NetworkManager/DnsManager[0m
[32m20-08-20 09:18:25 INFO (MainThread) [supervisor.host.sound] Update PulseAudio information[0m
Failed to load cookie file from cookie: No such file or directory
[32m20-08-20 09:18:59 INFO (SyncWorker_4) [supervisor.docker.interface] Clean homeassistant application[0m
[32m20-08-20 09:18:59 INFO (MainThread) [supervisor.homeassistant] Update pulse/client.config: /data/tmp/homeassistant_pulse[0m
[32m20-08-20 09:19:06 INFO (SyncWorker_2) [supervisor.docker.homeassistant] Start homeassistant homeassistant/raspberrypi3-homeassistant with version 0.114.2[0m
[32m20-08-20 09:19:06 INFO (MainThread) [supervisor.homeassistant] Wait until Home Assistant is ready[0m
[32m20-08-20 09:22:05 INFO (MainThread) [supervisor.homeassistant] Detect a running Home Assistant instance[0m
[32m20-08-20 09:22:05 INFO (MainThread) [supervisor.homeassistant] Successful run Home Assistant 0.114.2[0m
[32m20-08-20 09:22:05 INFO (SyncWorker_3) [supervisor.docker.interface] Cleanup images: ['homeassistant/raspberrypi3-homeassistant:0.114.0'][0m
[32m20-08-20 09:22:25 INFO (MainThread) [supervisor.api.security] /supervisor/logs access from cebe7a76_hassio_google_drive_backup[0m
[32m20-08-20 09:22:28 INFO (MainThread) [supervisor.api.security] /core/logs access from cebe7a76_hassio_google_drive_backup[0m
[32m20-08-20 09:22:58 INFO (MainThread) [supervisor.api.security] /supervisor/logs access from cebe7a76_hassio_google_drive_backup[0m

 ```
 ###### Home Assistant Core Logs:
 ```
 2020-08-20 11:19:41 WARNING (MainThread) [homeassistant.setup] Setup of recorder is taking over 10 seconds.
decrypt_verify(): found 24 bytes cleartext
decrypt_verify(): found 514 bytes cleartext
decrypt_verify(): found 52 bytes cleartext
decrypt_verify(): found 52 bytes cleartext
decrypt_verify(): found 52 bytes cleartext
decrypt_verify(): found 52 bytes cleartext
decrypt_verify(): found 232 bytes cleartext
decrypt_verify(): found 237 bytes cleartext
decrypt_verify(): found 237 bytes cleartext
decrypt_verify(): found 243 bytes cleartext
decrypt_verify(): found 237 bytes cleartext
decrypt_verify(): found 232 bytes cleartext
decrypt_verify(): found 232 bytes cleartext
decrypt_verify(): found 237 bytes cleartext
decrypt_verify(): found 237 bytes cleartext
decrypt_verify(): found 243 bytes cleartext
decrypt_verify(): found 237 bytes cleartext
decrypt_verify(): found 232 bytes cleartext
decrypt_verify(): found 232 bytes cleartext
decrypt_verify(): found 237 bytes cleartext
decrypt_verify(): found 237 bytes cleartext
decrypt_verify(): found 243 bytes cleartext
decrypt_verify(): found 237 bytes cleartext
decrypt_verify(): found 232 bytes cleartext
2020-08-20 11:20:00 WARNING (MainThread) [homeassistant.setup] Setup of default_config is taking over 10 seconds.
2020-08-20 11:20:45 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: default_config
2020-08-20 11:21:45 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: default_config
2020-08-20 11:22:00 ERROR (MainThread) [homeassistant.components.netatmo] Error during webhook registration - 400 - Bad request - Allowed ports for webhooks are 443 and 80. (21) when accessing 'https://api.netatmo.com/api/addwebhook'

 ```

**Comment by sabeechen** [comment]
Thanks for using the new bug report feature!  It helps a lot.

I think I might know why this happened, in the next release I'll add some handling for this error that gives up on the upload and starts over (this is Google saying it can't resume an upload).  In the meantime could you restart the addon?  I suspect it will fix itself then.

**Comment by razzietheman** [comment]
> Thanks for using the new bug report feature! It helps a lot.
> 
> I think I might know why this happened, in the next release I'll add some handling for this error that gives up on the upload and starts over (this is Google saying it can't resume an upload). In the meantime could you restart the addon? I suspect it will fix itself then.

yeah after a while don't really know what I did but it get back up and now works without problem for now

**Comment by sabeechen** [comment]
I've released a new version (0.101.2) that should automatically recover from this error without needing a restart.
